test/jdk/javax/management/remote/mandatory/notif/NotificationEmissionTest.java

```diff
@@ -32,6 +32,9 @@
  * @run clean NotificationEmissionTest
  * @run build NotificationEmissionTest
  * @run main NotificationEmissionTest 1
+
+ * @run main/othervm NotificationEmissionTest 1
```

Comment:
Lines 34 & 36: is it necessary to run NotificationEmissionTest twice with and w/o `othervm`?

Reply:
It's unnecessary, thanks.